Makefile

```diff
@@ -3,9 +3,9 @@ ARCH ?= $(shell uname -m)
 ifeq (${ARCH}, arm64)
 	ARCH = aarch64
 endif
+TARGETDIR = packages/${ARCH}
 
-MELANGE_DIR ?= ../melange
-MELANGE ?= ${MELANGE_DIR}/melange
+MELANGE ?= ../melange/melange
```

Comment:
why dont we just use $(which melange) here?

Reply:
Historical reasons.  Those of us using Melange natively are usually running it alongside the wolfi checkout.

Reply:
I'm just afraid this will be a bit unfriendly for new contributors. where they will need to have `melange` repo at the same level as `wolfi` repo. also, if their melange repo is staled, it could become problem for them, no?

Reply:
I agree it is a bit unfriendly, while also being convenient for using melange natively, although this PR makes that neither worse nor better.

I see two options:

A. Default to `MELANGE=$(which melange)` and have people using it natively override that with `make MELANGE=../melange/melange`
B. Default to `MELANGE=../melange/melange` and have people who do not have it checked it override it with `make MELANGE=$(which melange)`

My personal preference is B, but I didn't want to change too much UX in here without agreement from other folks.

There is a third possibility, i.e.

1. If `MELANGE` is set use that; else
2. If `../melange/melange` exists use that; else
3. If `$(which melange)` exists use that; else
4. Error out

Thoughts?

Reply:
Actually, as I think about it, every time something gets updated with the Makefile packages, I have to do a manual rebase. 

I would like to get this in as is; happy to address the `MELANGE` var separately afterwards.